{
    "containers": {
        "cna": {
            "affected": [
                {
                    "product": "VIOS ",
                    "vendor": "IBM",
                    "versions": [
                        {
                            "status": "affected",
                            "version": "3.1"
                        }
                    ]
                },
                {
                    "product": "AIX",
                    "vendor": "IBM",
                    "versions": [
                        {
                            "status": "affected",
                            "version": "7.1"
                        },
                        {
                            "status": "affected",
                            "version": "7.2"
                        },
                        {
                            "status": "affected",
                            "version": "7.3"
                        }
                    ]
                }
            ],
            "datePublic": "2022-03-01T00:00:00",
            "descriptions": [
                {
                    "lang": "en",
                    "value": "IBM AIX 7.1, 7.2, 7.3, and VIOS 3.1 could allow a non-privileged local user to exploit a vulnerability in the AIX kernel to cause a denial of service. IBM X-Force ID: 213076."
                }
            ],
            "metrics": [
                {
                    "cvssV3_0": {
                        "attackComplexity": "LOW",
                        "attackVector": "LOCAL",
                        "availabilityImpact": "HIGH",
                        "baseScore": 6.2,
                        "baseSeverity": "MEDIUM",
                        "confidentialityImpact": "NONE",
                        "exploitCodeMaturity": "UNPROVEN",
                        "integrityImpact": "NONE",
                        "privilegesRequired": "NONE",
                        "remediationLevel": "OFFICIAL_FIX",
                        "reportConfidence": "CONFIRMED",
                        "scope": "UNCHANGED",
                        "temporalScore": 5.4,
                        "temporalSeverity": "MEDIUM",
                        "userInteraction": "NONE",
                        "vectorString": "CVSS:3.0/I:N/UI:N/A:H/C:N/AC:L/PR:N/AV:L/S:U/E:U/RL:O/RC:C",
                        "version": "3.0"
                    }
                }
            ],
            "problemTypes": [
                {
                    "descriptions": [
                        {
                            "description": "Denial of Service",
                            "lang": "en",
                            "type": "text"
                        }
                    ]
                }
            ],
            "providerMetadata": {
                "dateUpdated": "2022-03-02T16:35:15",
                "orgId": "9a959283-ebb5-44b6-b705-dcc2bbced522",
                "shortName": "ibm"
            },
            "references": [
                {
                    "tags": [
                        "x_refsource_CONFIRM"
                    ],
                    "url": "https://www.ibm.com/support/pages/node/6560390"
                },
                {
                    "name": "ibm-aix-cve202138997-dos (213076)",
                    "tags": [
                        "vdb-entry",
                        "x_refsource_XF"
                    ],
                    "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/213076"
                }
            ]
        }
    },
    "cveMetadata": {
        "assignerOrgId": "9a959283-ebb5-44b6-b705-dcc2bbced522",
        "assignerShortName": "ibm",
        "cveId": "CVE-2021-38996",
        "dateReserved": "2021-08-16T00:00:00",
        "dateUpdated": "2022-03-02T16:35:15",
        "state": "PUBLISHED",
        "datePublished": "2022-03-02T16:35:15.526010Z"
    },
    "dataType": "CVE_RECORD",
    "dataVersion": "5.0"
}
